kafka/modis_kernels_kf.py

```diff
@@ -229,21 +230,22 @@ def get_mcd43_prior(mcd43_fstring, band):
                 gdal.Open(os.path.join(the_dir, "brdf_2016_b06.vrt")),
                 gdal.Open(os.path.join(the_dir, "brdf_2016_b07.vrt")))
 
-    output_plethora = create_output_file(g, the_dir, "test")
+    output_plethora = create_output_file(g, the_dir, "band{}".format(band))
     
-    kf = MODISKernelLinearKalman(modis_obs, days, output_plethora, [] )
+    kf = MODISKernelLinearKalman(modis_obs, days, output_plethora, [], the_band=band )
```

Comment:
Actually, this should be `the_band=band-1`, or else, subtract 1 from `self.the_band` in `_get_observations_timestep` method. I guess the 2nd option is preferred...